config/locales/in_person_proofing/en.yml

```diff
@@ -6,6 +6,16 @@ en:
         info: 'If your current address does not match the address on your ID, you will
           need to bring proof of your current address to the Post Office.'
         learn_more: Learn more
+      location:
+        location_button: 'Select'
+        location_step_about: 'If you are having trouble adding your ID, you may be able
+          to verify in person at a local United States Post Office in select
+          locations.'
+        post_office: 'Post Office ™'
```

Comment:
please forgive me, I've been out of the loop of recent design convos if this has already been hashed out, but from the screenshots, the repetition of "- POST OFFICE™" was super distracting visually. Are we required to show that?

Reply:
I don't know if there's been an update around that text that's not reflected in the Figma design but I can ask in the slack channel, thanks for pointing out that.

Reply:
agreed, good point @zachmargolis. IIRC, because we get this info from the USPS API, it was going to take more effort to remove that (and also use sentence case instead of all caps), so for MVP we were going to leave it, but feel free to correct me if I'm wrong!

Reply:
Nope, we are adding it ourselves, it's not coming from the API. So if we don't want it there, it's easy to remove.